### ISSUE TITLE
Upgrade to latest releases

### ISSUE THREAD

**Issue by lanhaoxiang** [gnosis/safe-infrastructure#26]
@Uxio0 Thanks a lot for your efficient work, is it possible to upgrade the docker images in this repo so we will be able to use the latest images directly

**Comment by Uxio0** [comment]
Yes, just send us a PR with our last dockerhub image versions for the 3 services and we will merge it. Remember to modify `README` and `.env.sample`.

We do it every few weeks as it's a manual process
